seqr/utils/elasticsearch/es_search.py

```diff
@@ -63,12 +65,12 @@ def __init__(self, families, previous_search_results=None, skip_unaffected_famil
             if len(self.samples_by_family_index) < 1:
                 raise Exception('Inheritance based search is disabled in families with no affected individuals')
 
-        self._indices = self.samples_by_family_index.keys()
+        self._indices = sorted(list(self.samples_by_family_index.keys()), reverse = True)
         self._set_index_metadata()
 
         if len(self.samples_by_family_index) != len(self.index_metadata):
             raise InvalidIndexException('Could not find expected indices: {}'.format(
-                ', '.join(set(self._indices) - set(self.index_metadata.keys()))
+                ', '.join(sorted(set(self._indices) - set(self.index_metadata.keys())))
```

Comment:
The purpose of adding `sorted()` here is to keep the expected test result to be stable.

Reply:
since you added `sorted` to the assignment for `self._indices` above, this should already be sorted here

Reply:
Although `self._indices` is sorted, the new set generated by the subtraction is still possible out of order.

Reply:
I saw the order of `set(self._indices) - set(self.index_metadata.keys())` changed from time to time in Python 3. But, since it is not ideal by adding `sorted()` for test purpose in production code, I updated the result assertion to accept different orders of the indices list.